watson/cli.py

```diff
@@ -1047,7 +1047,6 @@ def _final_print(lines):
                 project=style('project', u'{:>{}}'.format(
                     frame.project, longest_project
                 )),
-                pad=longest_project,
```

Comment:
Can explain this line removal a bit more?

Reply:
Travis wouldn't pass with it in place. See [this run](https://travis-ci.org/github/TailorDev/Watson/jobs/689886199). This is the actual error message:

```
flake8 run-test: commands[0] | flake8 --show-source watson/ tests/ scripts/

watson/cli.py:1045:13: F522 '...'.format(...) has unused named argument(s): pad
```

Reply:
(I'm happy to add it back, if that's what you think should happen)

Reply:
No, no, you are totally right (and so does flake8). I'm surprised that we did not see it before.